docker/dev.sh

```diff
@@ -151,3 +151,11 @@ docker run -itd \
   -v $(pwd):/opt/openoni:Z \
   -v $(pwd)/docker/data:/opt/openoni/data:z \
   open-oni:dev
+
+port=$(_port $APP_URL)
```

Comment:
I'm not sure I follow.  APP_URL isn't magic; you have to set its port yourself.  This is because what you expose via docker may not be what you use for a demo (for instance, I can port-forward from my docker VM's port 8080 to my host machine's port 80).

This should illustrate the parsing tossing a warning at you:

```bash
export APP_URL="http://localhost:8081"
export DOCKERPORT=8080
./docker/dev.sh
```

Note the big red warning.  Now change it:

```bash
export APP_URL="http://localhost:8080"
export DOCKERPORT=8080
./docker/dev.sh
```

Note that there is no longer a warning.